### PR TITLE
feat: add req in account linking so DFSP can store consent details

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: src/models/inbound/dfspLinking.interface.ts [hunk 1]
```diff
@@ -81,6 +81,12 @@ export interface BackendGetScopesResponse {
   scopes: tpAPI.Schemas.Scope[]
 }
 
+export interface BackendStoreValidatedConsentRequest {
```
Comment (lewisdaly) [inline src/models/inbound/dfspLinking.interface.ts:84]: Is there a reason you're not also sending the credential along here?

**Review by lewisdaly — COMMENTED** [review]
Looks good to me as far as I understand this service... just 1 comment.

File: src/models/inbound/dfspLinking.interface.ts (hunk 1 above)
Comment (kleyow) [reply src/models/inbound/dfspLinking.interface.ts:84]: I can. Do you want me to just send the whole credential?

Comment (lewisdaly) [reply src/models/inbound/dfspLinking.interface.ts:84]: Yeah I think so... that way the DFSP has it for auditing purposes or if they want to check the credential themselves.

**Review by lewisdaly — APPROVED** [review]

**Pull request merged**